JASP-Engine/JASP/R/LSbinomialestimation.R

```diff
@@ -956,8 +956,8 @@ LSbinomialestimation   <- function(jaspResults, dataset, options, state = NULL){
                                                 options[["plotsIterativeBF"]])
           }
           
-          if (all(is.na(tempCIPP[1:2]))){
-            tempInt <- "∅"
+          if (all(is.na(temp_CIPP[1:2]))){
+            temp_int <- "\u2205"
```

Comment:
The `else` clause mentions `tempInt` -- I don't think we want this variable rename

Reply:
Huh, maybe someone changed the name in between my first attempt at unicode fix and now, I'll fix it.